Jenkinsfile

```
@@ -7,6 +7,7 @@ hose {
     RELEASETIMEOUT = 30
     FOSS = true
     REPOSITORY = 'common-utils'
+    DEPLOYONPRS = true
```

This should be setted to false on final version.

```suggestion
    DEPLOYONPRS = false
```

---

Yes, it is just for testing purposes.
It will be removed before merging.

---

Line removed :+1: